client/app/scripts/superdesk-users/views/user-preferences.html

```diff
@@ -30,6 +33,34 @@
                     <span>{{ datelinePreview | previewDateline:datelineSource}} </span>
                 </div>
             </li>
+
+            <li class="fieldset">
+              <div class="legend">
+                {{ :: preferences['categories:preferred'].category }}
+              </div>
+
+              <div sd-info-item>
+                  <label translate>
+                    {{ :: preferences['categories:preferred'].label }}
+                  </label>
+                  <label translate>
+                    By checking the categories you are interested in, the system
+                    will only display them in a menu for setting the content
+                    item's categories (along with any of its existing categories).
+                  </label>
+
+                  <div ng-repeat="cat in categories track by cat.qcode">

```

Comment:
Hm, the checkboxes were explicitly mentioned when discussing this feature, nevertheless we can of course change that.
There is a particular advantage with checkboxes, though, and that is the users immediately see which categories are not selected, allowing them to select them. With the sd-typeahead, on the other hand, they would have to remember the category name in order to type its name and select it.

I'm open to different arguments here, and can change the UI widget if deemed necessary.